packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -33,7 +20,31 @@ import { name, version } from '../package'
  * - packages/create-redwood-app/src/create-redwood-app.js (this file)
  *
  */
+import chalk from 'chalk'
+import checkNodeVersion from 'check-node-version'
+import { prompt } from 'enquirer'
+import execa from 'execa'
+import fs from 'fs-extra'
+import { Listr, figures } from 'listr2'
+import terminalLink from 'terminal-link'
+import { hideBin } from 'yargs/helpers'
+import yargs from 'yargs/yargs'
+
+import { name, version } from '../package'
+
+import { startTelemetry, shutdownTelemetry } from './telemetry'
 ;(async () => {
+  //
+
+  // Telemetry
+  if (
+    !process.argv.includes('--no-telemetry') && // Must include '--no-telemetry' exactly because we want to do this check before any yargs. TODO: Communicate this on cmd help
```

Comment:
This comment is a little too long to be on the same line as the code:

```suggestion
    // Must include '--no-telemetry' exactly because we want to do this check before any yargs.
    // TODO: Communicate this on cmd help
    !process.argv.includes('--no-telemetry') &&
```